landing.css

```diff
@@ -0,0 +1,126 @@
+@import "./base.css";
+@import "./utility.css";
+
+/* Navigation bar */
+
+.brand-name{
+    font-size: 2rem;
+    padding-bottom: 0.5rem;
+}
+.brand-logo{
+    margin-left: 1.6rem;
+    margin-top:1.2rem;
+}
+.nav-bar-hero{
+    display:flex
+}
+.nav-header{
+    background-color: var(--secondary-color) ;
+    display: flex;
```

Comment:
A good approach would be to use helper classes for repetitive but useful properties like flex, justify, align
Then you can directly use the classnames in your other projects, shortening your CSS lines as well

Reply:
> A good approach would be to use helper classes for repetitive but useful properties like flex, justify, align Then you can directly use the classnames in your other projects, shortening your CSS lines as well

Noted and implemented